easybuild/easyconfigs/o/OpenSSL/OpenSSL-3.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'EB_OpenSSL_wrapper'
+
+name = 'OpenSSL'
+version = '3'
+minimum_openssl_version = '3.0.0'
+
+homepage = 'https://www.openssl.org/'
+description = """The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,
+ and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) 
+ protocols as well as a full-strength general purpose cryptography library. """
+
+toolchain = SYSTEM
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+    ('Perl', '5.38.0'),
+]
+
+# This easyconfig will wrap the OpenSSL installation in the host system.
+# If the system provides the required binary, header files, and libraries for
+# this version of OpenSSL, the installation directory of this module will be
+# populated with symlinks to the system files. The minimum required version of
+# OpenSSL can be finely controled with 'minimum_openssl_version' (defaults to
+# easyconfig version).
+# If the host system does not have this version of OpenSSL (or with the option
+# wrap_system_openssl = False), EasyBuild will fall back to the following
+# component list, which will be build and installed as usual.
+
+components = [
+    (name, '3.2.0', {
```

Comment:
@SebastianAchilles Let's bump this to `3.2.1`?

Reply:
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/19623/commits/56793bccdb8f0ad1b9b575e8e549ccafd7eb329e